resources/views/components/organisms/_o-table-of-contents.blade.php

```diff
@@ -1,21 +1,30 @@
 <div class="o-accordion{{ (isset($variation)) ? ' '.$variation : '' }}" data-behavior="accordion">
     @foreach ($items as $item)
+        @php
+            if (isset($currentArticle)) {
+                $isExpanded = $currentArticle->parent === $item || $item->present()->isArticleInTree($item->children, $currentArticle);
```

Comment:
With the second parameter of `isArticleInTree` removed, refactor this call to `$currentArticle->present()->isArticleInTree($item->children);`